app/helpers/external_pledges_helper.rb

```diff
@@ -1,13 +1,14 @@
 # Functions to populate dropdowns related to external pledges.
 module ExternalPledgesHelper
   def external_pledge_source_options
-    Rails.configuration
-         .external_pledges['funds']
-         .reject { |fund| fund == FUND_FULL }
+    standard_options = ['Clinic discount', 'Other funds (see notes)']
+    Config.find_or_create_by(config_key: 'external_pledge_source').options
+          .push(*standard_options)
+          .uniq
   end
 
   def available_pledge_source_options_for(patient)
     used_sources = patient.external_pledges.map(&:source)
-    (external_pledge_source_options - used_sources).unshift(nil)
+    (external_pledge_source_options - used_sources)
```

Comment:
Was the `unshift(nil)` placing nil on the array as the default value? What changed that we don't want that now?